Altis_Life.Altis/core/functions/fn_startLoadout.sqf

```diff
@@ -0,0 +1,51 @@
+/*
+    File: fn_startLoadout.sqf
+    Author: Casperento
+
+    Description:
+    Loads a custom loadout on player
+*/
+private _loadOutCfg = getArray(missionConfigFile >> "Loadouts" >> str(side player) >> "loadOut");
```

Comment:
it's probably best to swap to using the `M_CONFIG` macro and include the macros to keep things consistent.